verification/curator-service/api/src/controllers/cases.ts

```diff
@@ -136,11 +167,20 @@ export default class CasesController {
                 });
             });
 
-            await User.findOneAndUpdate(
-                {
-                    _id: user._id,
-                },
-                { $push: { downloads: { timestamp: new Date() } } },
+            await User.findByIdAndUpdate(
+                user._id,
+                { $push: { downloads: {
+                    timestamp: new Date(),
+                    format: req.body.format,
+                    query: req.body.query,
```

Comment:
Should I leave `format` and `query` out here? Or make them `csv` and `{}`?

Reply:
I would say pass on what's in the req, but check in swagger if they're required here and supply defaults if not.

Reply:
They're optional fields for backwards compatibility. Omitting them here.